nchelpers/__init__.py

```diff
@@ -1637,27 +1642,27 @@ def _cmor_type_filename_components(
             )
         }
 
+        if self.is_multi_year:
+            components.update(
+                frequency=self.frequency
+            )
+        else:
+            # Regarding how the 'mip_table' component is defined here,
+            # see the discussion in section titled "MIP table / table_id" in
+            # https://pcic.uvic.ca/confluence/display/CSG/PCIC+metadata+standard+for+downscaled+data+and+hydrology+modelling+data
+            # Specifically, we do not consult the value of the attribute
+            # ``table_id`` because it is too limited for our needs. Instead we
+            # map the file's time resolution to a value.
+            components.update(
+                mip_table=tres_to_mip_table and
+                          tres_to_mip_table.get(self.time_resolution, None)
+            )
+
```

Comment:
Good. You've pulled the multi-year case into a case independent of other conditions. Is this what you are thinking here, i.e., that multi-year is independent of all other ways of categorizing the file?

Reply:
Oh, I see by your comments in the PR above that you did. Good.